components/authz-service/server/v2/rules_property_test.go

```diff
@@ -33,25 +37,25 @@ func TestCreateRuleProperties(t *testing.T) {
 			api.ProjectRuleConditionAttributes_POLICY_GROUP,
 			api.ProjectRuleConditionAttributes_POLICY_NAME,
 		),
-		"Values": gen.SliceOf(gen.AnyString()),
+		"Values": gen.SliceOf(gen.UnicodeString(graphicRange)),
 	})
 	conditionsGenEvent := gen.StructPtr(reflect.TypeOf(&api.Condition{}), map[string]gopter.Gen{
 		"Type": gen.OneConstOf(
```

Comment:
These `Type` keys should be `Attribute` now

Reply:
...Surprised this worked then because it's actually doing database inserts.